src/utils/regex.js

```diff
@@ -1,5 +1,5 @@
 export default {
-  address: /^[1-9]\d{0,19}L$/,
+  address: /^lsk[a-z0-9]{0,38}$/,
```

Comment:
I assume this is a temporary change and will be replaced by Lisk Elements.

Reply:
Yes and we need to update it all across the repo. This is only temporary.